### PR TITLE
Add section about inspecting compressed data

### PR DESCRIPTION
**Pull request by uglide**
## Purpose of this pull request

This pull request (PR) adds a section on how to easily inspect compressed Session and Cache data stored in Redis.
 This task is quite common for Magento developers and this is now supported out of the box in Open Source Redis GUI
More details are here: https://github.com/uglide/RedisDesktopManager/issues/5028

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.4/config-guide/redis/redis-session.html
- https://devdocs.magento.com/guides/v2.4/config-guide/redis/redis-pg-cache.html

### CODE REVIEW

File: src/_includes/config/redis-verify.md [hunk 1]
```diff
@@ -50,6 +50,11 @@ If you use Redis for page caching, you'll see output similar to the following:
 ... more ...
 ```
 
+### Inspect Compressed Sesssion data and Page Cache with GUI tool
```
Comment (BarnyShergold) [inline src/_includes/config/redis-verify.md:53]:
```suggestion
### Inspect Compressed Session data and Page Cache with GUI tool
```

**Review by BarnyShergold — CHANGES_REQUESTED** [review]
One small change
Plus @dobooth is it okay to suggest external application to be installed?

**Review by dobooth — APPROVED** [review]

**Comment by dobooth** [comment]
It's ok to recommend a 3rd party tool.

**Comment by dobooth** [comment]
@magento import pr to magento-devdocs/devdocs

**Comment by m2-github-services** [comment]
@dobooth the Pull Request is successfully imported.

**Pull request merged**